src/clarity-angular/nav/_nav.clarity.scss

```diff
@@ -49,8 +49,7 @@ $clr-active-nav-link-shadow: 0 -3px 0 $action-blues-dark-midtone inset;
             }
 
             &:hover,
-            &.active,
-            &:focus {
```

Comment:
also we can add a `outline-offset: -3px` or some other -ve offset to make sure that the focus state is within the tab component